docs/about/licensing.md

```diff
@@ -5,31 +5,38 @@ sidebar_label: Amplication’s licensing model
+## Open Source and Enterprise Offerings
 
-Amplication can now meet the needs of individual developers and startups and provide the advanced features, robustness, and scale required by enterprises through its Free and Enterprise Edition:
+### Free Edition
 
-- **Free Edition** – contains Amplication's core functionality and continues to be licensed under the [Apache 2.0 license](https://github.com/amplication/amplication/blob/master/LICENSE).
-- **Enterprise Edition** - add advanced operations, security, and commercially orientated features. The Enterprise edition is built on top of the Free edition and the additional functionality will be supplied as source-available. While users can view and download the code, ownership of the code remains with Amplication, meaning users cannot resell it. Use of the Amplication Enterprise edition is subject to the [Amplication Terms of Service](https://amplication.com/terms).
+- The Free Edition of Amplication includes our core functionality, based on Node.js. This edition is available under the [Apache 2.0 license](https://github.com/amplication/amplication/blob/master/LICENSE) and remains free for use on our managed service at [amplication.com](https://amplication.com).
+- Developers can freely access, modify, and distribute the source code available in this edition, fostering a collaborative and open development environment.
 
-## **Amplication’s Continuing Commitment to Open Source**[](https://docs.amplication.com/about/about/licensing/#amplications-continuing-commitment-to-open-source)
+### Enterprise Edition
 
-- Amplication remains fully committed to open source and promises to be a good steward of its open-source project, by ensuring that it continues to evolve with powerful new features. As we extend our Enterprise edition, we will ensure that this does not conflict with our open-source project.
-- When a feature has been contributed by members of the Amplication community, we won't move that feature to the Enterprise edition.
-- If a feature is planned for Free and the Enterprise edition, it will be released simultaneously in all. We won't introduce features into the Free Edition codebase with a fixed delay.
-- We will always make it clear what is proprietary and what is open-source code.
-- We will keep all features including the Enterprise Edition features for free on Amplication Cloud for the open-source community, individual developers, and small-scale projects.
+- The Enterprise Edition extends the Free Edition by incorporating advanced operational and security features. This edition introduces support for .NET applications, offering robust, scalable backend solutions that leverage popular .NET frameworks such as ASP.NET Core and Entity Framework Core.
```

Comment:
I think we need to mention node.js here as well. Something like-
"The Enterprise Edition extends the Node.js Free Edition by..."